packages/@react-spectrum/utils/src/styleProps.ts

```diff
@@ -232,19 +232,26 @@ export function passthroughStyle(value) {
   return value;
 }
 
-const boxAlignmentStyleProps: StyleHandlers = {
-  justifyItems: ['justifyItems', passthroughStyle],
-  justifyContent: ['justifyContent', passthroughStyle],
-  alignItems: ['alignItems', passthroughStyle],
-  alignContent: ['alignContent', passthroughStyle],
-  ...baseStyleProps
-};
+// Normalize 'start' and 'end' alignment values to 'flex-start' and 'flex-end'
+// in flex containers for browser compatibility.
+function flexAlignValue(value) {
+  if (value === 'start') {
```

Comment:
any test we can do to see if 'start' is supported before changing it to 'flex-start'?

Reply:
we could, but is it worth it?